notebooks/3_KernelMethods.ipynb

```diff
@@ -510,7 +536,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Our regularization is {{regularization*lambda_max}}."
+    "Our regularization is {{regularization*lambda_max}}.\n",
+    "\n",
+    "**Note**: To display the value of the regularization parameter, you should [enable the jupyter contrib nbextension](https://stackoverflow.com/questions/52812231/print-variable-in-jupyter-notebook-markdown-cell-python)."
```

Comment:
This note should go at the beginning of the tutorials, not here.

Reply:
I will move it to the beginning.